torch/csrc/lazy/core/lazy_graph_executor.cpp

```diff
@@ -679,13 +679,37 @@ std::vector<Value> LazyGraphExecutor::CollectRoots(
   return roots;
 }
 
-std::vector<BackendDataPtr> LazyGraphExecutor::FetchTensorData(
+void LazyGraphExecutor::ExtractIRAndPrepareXlaData(
```

Comment:
Can you omit `Xla` in the name? Change it to `ExtractIRAndPrepareTensorData`

Reply:
bump

Reply:
yeah, let me make the change